worlds/sc2/docs/en_Custom Mission Orders.md

```diff
@@ -0,0 +1,536 @@
+
+## Shared options
+
+All the options below are listed with their defaults.
+
+---
+```yaml
+# For campaigns
+required: true
+# For layouts and missions
+required: false
+```
+For campaigns, this determines whether the campaign is required to beat the world. If you turn this off for every campaign, the campaigns with the highest order are automatically marked as required.
+
+For layouts and missions, this determines whether the layout or mission is required to beat the campaign they are in. If no missions or layouts are marked as required, the layouts of the highest order are automatically marked as required.
+
+---
+```yaml
+order: 0
+```
+Applies to campaigns and layouts. Can be any integer value.
+
+The order determines where the campaign or layout is placed relative to other campaigns or layouts. Lower orders appear earlier and ties are considered to be parallel. For layouts, the order is only relative to other layouts within the same campaign.
+
+The order also determines where the campaign or layout is placed in the client. Lower-order campaigns are higher up, and lower-order layouts are further left. Ties are broken randomly.
+
+---
+
+```yaml
+unlock_count: -1 
+```
+Applies to campaigns and layouts. Must be bigger than or equal to -1.
+
+Missions also have a `unlock_count` option, but it works differently for them, and is covered later.
+
+This determines how many campaigns or layouts of the next-lowest order need to be beaten to access this campaign or layout. Negative values, including the default, mean that every previous-order campaign or layout is required.
+
+Next-lowest and previous order here only count orders that are defined in the YAML. If you want a campaign or layout to have a specific order but not use this access rule, set `unlock_count: 0` on it.
+
+The following is an example of these two options:
+```yaml
+  custom_mission_order:
+    Campaign A:
+      order: 0 # This is the default and could be omitted
+      global:
+        type: column
+        size: 3
+      Layout A-1:
+        order: -5 # Negative orders are allowed
+      Layout A-2:
+        order: 1
+      Layout A-3:
+        order: 1
+      Layout A-4:
+        order: 2
+        unlock_count: 1
+    Campaign B:
+      order: 1
+      Layout B-1:
+        order: -3
+        type: column
+        size: 3
+```
+This would result in the following visual layout in the client:
+```
+                           Campaign A
+  Layout A-1   |  Layout A-2   |  Layout A-3   |  Layout A-4
+---------------------------------------------------------------
+ Mission A-1-0 | Mission A-2-0 | Mission A-3-0 | Mission A-4-0
+ Mission A-1-1 | Mission A-2-1 | Mission A-3-1 | Mission A-4-1
+ Mission A-1-2 | Mission A-2-2 | Mission A-3-2 | Mission A-4-2
+
+  Campaign B
+  Layout B-1
+---------------
+ Mission B-1-0
+ Mission B-1-1
+ Mission B-1-2
+```
```

Comment:
Note markdown has [built-in tables](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables) (though I'm not sure the AP converter to HTML supports it; I'd expect it to).

```
|   Layout A-1  |  Layout A-2   |  Layout A-3   |  Layout A-4 |
| ------------- | ------------- | ------------ | ---------------- |
| Mission A-1-0 | Mission A-2-0 | Mission A-3-0 | Mission A-4-0 |
| Mission A-1-1 | Mission A-2-1 | Mission A-3-1 | Mission A-4-1 |
| Mission A-1-2 | Mission A-2-2 | Mission A-3-2 | Mission A-4-2 |
```
Renders as:

|   Layout A-1   |  Layout A-2   |  Layout A-3   |  Layout A-4 |
| -------------- | -------------- | ------------ | ---------------- |
| Mission A-1-0 | Mission A-2-0 | Mission A-3-0 | Mission A-4-0 |
| Mission A-1-1 | Mission A-2-1 | Mission A-3-1 | Mission A-4-1 |
| Mission A-1-2 | Mission A-2-2 | Mission A-3-2 | Mission A-4-2 |

If the webhost doesn't support it, you should be able to get tables with HTML tags `<table>`, `<tr>`, and `<th>`